index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/0eaab31a39.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./css/style.css">
+    <title>Apple homepage</title>
+</head>
+
+<body>
+   <div class="grid-container">
+       <main class="main flex-col">
+           <nav class="nav-bar flex-row">
+                <span class="nav-item"><i class="fab fa-apple"></i></span>
+                <span class="nav-item"><a href="#">Store</a></span>
+                <span><a href="#">Mac</a></span>
+                <span><a href="#">iPod</a></span>
+                <span><a href="#">iPhone</a></span>
+                <span><a href="#">iPad</a></span>
+                <span><a href="#">iTunes</a></span>
+                <span><a href="#">Support</a></span>
+                <span class="mag-span flex-row">
+                    <span class="mag">
+                        <img src="./images/mag.png" alt="">
+                    </span>
+                    <input type="text" name="search" class="nav-search-input">
+                </span>
+            </nav>
+           <div class="titles flex-col">
+                <a href="#"><img class="h1-img" src="./images/banner-h1.png" alt=""></a>
+                <a href="#"><img class="h2-img" src="./images/banner-h2.png" alt=""></a>
+                <a href="#"><img class="h3-img" src="./images/banner-h3.png" alt=""></a>
+            </div>
```

Comment:
- [ ] As was pointed out to you in the previous review, this section is still not displaying as it should. C
Please compare your clone with the source page and fix it: 

| Source Page	| Clone Page |
|:---:|:---:|
|![Screen Shot 2020-08-30 at 6 57 17 PM](https://user-images.githubusercontent.com/48804258/91664933-a0dc5180-eaf2-11ea-837d-1c7d02d079ad.png)|![Screen Shot 2020-08-30 at 6 58 07 PM](https://user-images.githubusercontent.com/48804258/91664951-bea9b680-eaf2-11ea-877c-ed059b223726.png)|